LEAF_Request_Portal/admin/index.php

```diff
@@ -54,10 +54,11 @@
 }
 
 $main = new Smarty;
-$t_login = new Smarty;
-$t_menu = new Smarty;
-$o_login = '';
-$o_menu = '';
+//nav (menu) is handled with vue component.  login does not look like it is still used
```

Comment:
Are we removing these commented out code?  We shouldn't leave it in if no longer used.